db/seeds/baseline_rates/addl_rates.rb

```diff
@@ -1,58 +1,62 @@
-class AdditionalRates
-  def self.parse(sheet, daterange)
-    shorthauls = Array.new
-    full_packs = Array.new
-    full_unpacks = Array.new
-    (3..sheet.last_row).each do |i|
-      # Shorthaul rates
-      if sheet.cell(i, 1) == 999
-        cwtm_min, cwtm_max = parse_shorthaul(sheet.cell(i, 4))
-        shorthauls << [Range.new(cwtm_min, cwtm_max), BigDecimal.new(sheet.cell(i, 5), 8), daterange]
+module Seeds
+  module BaselineRates
+    class Domestic400NG
```

Comment:
This class name is also defined in `db/seeds/baseline_rates/domestic_400ng.rb`. Is that correct?

Reply:
Looks like the class defined in `db/seeds/baseline_rates/domestic_400ng.rb` includes public methods where this file includes only `private` methods. Should these files be combined?